File: java/PE028.java
public class PE028 {
    public static void main(String[] args) {
        int sum = 1;
        int a = 1;
        int b = 2;
        for(int i = 1; i <= 500; i++) {
            for(int j = 1; j <= 4; j++) {
                a += b;
                sum += a;
            }
            b += 2;
        }
        System.out.println(sum);
    }
}

File: cpp/PE028.cpp
#include <iostream>
using namespace std;

int main() {
    int i, j, a = 1, sum = 1, step;
    for(i = 1; i <= (1001-1)/2; i++){
        step = 2*i;
        for(j = 1; j <= 4; j++){
            a += step;
            sum += a;
        }
    }
    cout << sum << endl;
    return 0;
}
